go-controller/pkg/ovn/controller/egressservice/egressservice_zone.go

```diff
@@ -522,12 +522,15 @@ func (c *Controller) repair() error {
 	}
 
 	// update caches after transaction completed
+	var allLocalV4Eps []string
 	for key, v4ToAdd := range svcKeyToLocalConfiguredV4Endpoints {
 		c.services[key].v4LocalEndpoints.Insert(v4ToAdd...)
+		allLocalV4Eps = append(allLocalV4Eps, v4ToAdd...)
```

Comment:
nit: iirc there might be duplicates here (if some services back the same ep), is it worth it to have this as a set (does an address set already handle duplicate entries?)

Reply:
> (does an address set already handle duplicate entries?)

it does: https://github.com/ovn-org/ovn-kubernetes/blob/c973d615cbfc2e68cc1dc2cb14e2a836a05f88cb/go-controller/pkg/ovn/address_set/address_set.go#L500